packages/expo-dev-launcher/package.json

```diff
@@ -28,7 +28,8 @@
   "license": "MIT",
   "homepage": "https://docs.expo.io",
   "dependencies": {
-    "@expo/config-plugins": "^1.0.32"
+    "@expo/config-plugins": "^1.0.32",
+    "expo-updates-interface": "~1.0.0"
```

Comment:
We don't want to have this interface here.

Reply:
Do we want it in the expo-dev-client package instead? Either way, I'm curious why we don't want this here; the code will not compile without this dependency.

Reply:
Yes, we want to add it to the expo-dev-client package instead. But also to the `dependencies.js` file to link it automatically to the react-native project. See https://github.com/expo/expo/blob/6d31ade729586695bf9aaf68c04367aa4d68190b/packages/expo-dev-client/dependencies.js#L3-L13

We don't want to have it here cause even with it, it won't compile. The react-native won't link it.